index.bs

```diff
@@ -2085,10 +2086,10 @@ Consensus</h4>
 	Note: Chairs have substantial flexibility
 	in how they obtain and assess consensus among their groups.
 	Unless otherwise constrained by charter,
-	they may use modes including but not limited to explicit calls for consensus,
+	they can use modes including but not limited to explicit calls for consensus,
 	polls of participants,
 	“lazy consensus” in which lack of objection after sufficient notice is taken as assent;
-	or they may also delegate and empower a document editor
+	or they can also delegate and empower a document editor
```

Comment:
I'm not a huge fan of and/or in general, but I understand wanting to avoid the sense of exclusive alternative. For this sentence, I feel that just deleting the word `or` adequately deals with the issue.